cloudbuild.yaml

```diff
@@ -0,0 +1,52 @@
+timeout: 9000s # 2.5 hours
+logsBucket: 'gs://$_LOGS_BUCKET_NAME'
+steps:
```

Comment:
@shubha-rajan @U8NWXD could one of you try changing the machineType here according to https://cloud.google.com/cloud-build/docs/build-config#options ? This seems to be what is recommended here: https://cloud.google.com/cloud-build/docs/speeding-up-builds#using_custom_virtual_machine_sizes

Reply:
Also, for the record, so far the cost for Cloud Build is $0.29 incurred on Jan 5. (Mentioning this because we should track it again with a higher build type.)